fs/dafsclient/rmtclient.cpp

```diff
@@ -411,17 +411,22 @@ void receiveDaFsBuffer(ISocket * socket, MemoryBuffer & tgt, unsigned numtries,
 {
     sRFTM tm(dafsMaxReceiveTimeMs);
     size32_t gotLength = receiveDaFsBufferSize(socket, numtries,tm.timemon);
-    if (gotLength) {
+    Owned<IException> exception;
+    if (gotLength)
+    {
         size32_t origlen = tgt.length();
-        try {
-            if (gotLength>maxsz) {
+        try
+        {
+            if (gotLength>maxsz)
+            {
                 StringBuffer msg;
                 msg.appendf("receiveBuffer maximum block size exceeded %d/%d",gotLength,maxsz);
                 PrintStackReport();
```

Comment:
not new, but think should remove this PrintStackReport() line, as the exception handling is going to log it again, and will be able to tell it was because thrown here.

Reply:
Agreed